src/fiskaltrust.ifPOS/v1/it/Models/Info.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace fiskaltrust.ifPOS.v1.it
+{
+
+    /// <summary>
+    /// Details about the operational status of the printer or server
+    /// </summary>
+    [DataContract]
+    public class Info
```

Comment:
I am not sure about the name. Would love to have sth more meaningful that we can use for the various cases that we will support.